### DISCUSSION TITLE
Currency datatype - error with decimal fraction [Apr 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1804] [Discord Threads]
> 🙋‍♂️ **softinio**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@460833082002374656> - its a bug. we will look into it   
#### Created an issue here    
#### https://github.com/nocodb/nocodb/issues/1647  
#  
> 🙋‍♂️ **softinio**  
#### <@857998597474484295> thank you for creating issue and quick help  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/963321317753495563)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
